storybook/src/components/annotation/annotation.core-template.ts

```diff
@@ -1,13 +1,92 @@
 import { Annotation } from "dso-toolkit";
-import { html, TemplateResult } from "lit-html";
+import { html } from "lit-html";
 
-import { ComponentImplementation, Templates } from "../../templates";
+import { ComponentImplementation } from "../../templates";
+import { AnnotationActiveChangeEvent, DsoAnnotationActiviteitCustomEvent } from "@dso-toolkit/core";
 
-export const coreAnnotation: ComponentImplementation<Annotation<TemplateResult>> = {
+export const coreAnnotation: ComponentImplementation<Annotation> = {
   component: "annotation",
   implementation: "core",
-  template: ({ annotationButtonTemplate, annotationOutputTemplate }: Templates) =>
-    function annotation({ annotationButton, annotationOutput }: Annotation<TemplateResult>) {
-      return html`${annotationButtonTemplate(annotationButton)}${annotationOutputTemplate(annotationOutput)}`;
+  template: () =>
+    function annotation(annotation: Annotation) {
+      if (annotation.type === "activiteit") {
+        const {
+          locatieNoemers,
+          naam,
+          regelKwalificatie,
+          symboolCode,
+          type,
+          active,
+          dsoActiveChange,
+          gewijzigdeLocatie,
+          regelKwalificatieVoorzetsel,
+          wijzigactie,
+        } = annotation;
+
+        return html`<dso-annotation-activiteit
+          .locatieNoemers=${locatieNoemers}
+          .naam=${naam}
+          .regelKwalificatie=${regelKwalificatie}
+          .type=${type}
```

Comment:
type wordt wel gezet, maar ik zie in `packages/core/src/components/annotation/annotation-activiteit.tsx` geen `@Prop()`. Dit geldt ook voor de andere 3.

Mis ik misschien iets?

Wordt wel onder properties in de browser getoond:
<img width="1231" alt="Screenshot 2024-05-31 at 09 47 16" src="https://github.com/dso-toolkit/dso-toolkit/assets/4646174/6e9b2263-c808-45f9-855c-5c14d81c5068">

Reply:
Done!